openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -1287,19 +1287,19 @@ def layer_structure_print(info: dict) -> None:
                                 )
                                 tf_layers_dict[layer_id] = \
                                     tf.maximum(0.0, inp) + \
-                                        tf.minimum(0.0, temp_alpha * inp)
+                                        tf.minimum(0.0, temp_alpha.transpose(1, 2, 0) * inp)
 
                             elif wr_config[layer_id]['replace_mode'] == 'insert_after':
                                 inp = tf.maximum(0.0, tf_layers_dict[get_tf_edges_from(tf_edges, layer_id, 0)]) + \
-                                    tf.minimum(0.0, temp_alpha * tf_layers_dict[get_tf_edges_from(tf_edges, layer_id, 0)])
+                                    tf.minimum(0.0, temp_alpha.transpose(1, 2, 0) * tf_layers_dict[get_tf_edges_from(tf_edges, layer_id, 0)])
```

Comment:
Aborts when `alpha_len` is 2. Therefore, I think we need to process branch `elif alpha_len >= 2:`.